src/app/online-purchase/online-purchase.component.html

```diff
@@ -0,0 +1,47 @@
+<h3 class="success-message">{{msg}}</h3>
+<div class="container">
+    <h4>Online Purchase</h4>
+    <!-- Trigger the modal with a button -->
+    <label class="switch">
+        <input type="checkbox"  data-toggle="modal" data-target="#myModal" (change)="onclick()">
+        <span class="slider round"></span>
+    </label>
+    <!-- Modal -->
+    <div class="modal fade" id="myModal" role="dialog">
+        <div class="modal-dialog modal-lg">
+            <div class="modal-content">
+                <div>
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">×</button>
+                        <h3 class="warning-msg">
+                          <span *ngIf="labelFlag">{{labelforActivateMsgHeader}}</span>
+                          <span *ngIf="!labelFlag">{{labelfoDeactivateMsgHeader}}</span>
+                        </h3>
```

Comment:
Try to use if else rather than using two if

Reply:
done using if else.